frontend_tests/node_tests/typing_status.js

```diff
@@ -263,6 +264,7 @@ run_test("basics", () => {
     // test that we correctly detect if worker.get_recipient
     // and typing_status.state.current_recipient are the same
 
+    compose_state.get_message_type = () => "private";
     compose_pm_pill.get_user_ids_string = () => "1,2,3";
     typing_status.state.current_recipient = typing.get_recipient();
 
```

Comment:
I haven't added/modified tests here as it's intended to test `typing_status.js` to which this PR makes no changes. But we make use of this module to send start/stop requests from client which only extends the possible values for `status.current_recipient` to also be an object consisting `stream_id` and `topic`. since we only check if current recipient is equal to previous recipient, adding tests to include recipient format of object:`stream_id, topic` will be redundant and won't add much value I think.

Will add few tests to also include that format for `current_recipient` if you think it's necessary.